pkg/apis/core/v1alpha1/types_utils.go

```diff
@@ -30,7 +30,7 @@ const (
 // https://github.com/kubernetes/kubernetes/issues/55890
 // https://github.com/kubernetes-sigs/cluster-api/issues/137
 type ProviderConfig struct {
-	runtime.RawExtension `json:",inline"`
+	runtime.RawExtension `json:",inline" protobuf:"bytes,1,opt,name=rawExtension"`
```

Comment:
Is this correct (especially `name=rawExtension`)? Why is not it not done for all inline types (e.g., `TypeMeta`)?